### DISCUSSION TITLE
Welcome to List-of-all-Research-Internship-Program-for-IIT-and-NITs Discussions!

### DISCUSSION THREAD

**Discussion by vvignesh17** [vvignesh17/List-of-all-Research-Internship-Program-for-IIT-and-NITs#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * If anyone finds any new Internship, add it to the list and also if someone finds any Deadlines wrong,  Update it according to the current year deadlines.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
 

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Nandan-N** [comment]
Perfect.
